website_event_crm/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "maintainers": ["Yajo"],
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["crm_event", "website_event"],
+    "depends": ["crm_event", "website_event", "sale_stock"],
```

Comment:
Why this new dependency?

Reply:
Sorry, I had it for local testing purposes